src/Icon/Icon.js

```diff
@@ -25,12 +24,10 @@ Icon.displayName = 'Icon';
 Icon.propTypes = {
     glyph: PropTypes.string.isRequired,
     className: PropTypes.string,
-    clickHandler: PropTypes.func,
     size: PropTypes.oneOf(ICON_SIZES)
 };
 
 Icon.propDescriptions = {
-    clickHandler: 'Callback function when user clicks on the component.',
```

Comment:
Needed to standardize on using `onClick` so it gets wired up properly if an `Icon` is used as the `control` for a `Popover`.